src/javaRosa.js

```diff
@@ -1834,6 +1834,9 @@ define([
                     if (hasText && !value.autoId) {
                         data[name] = value.id;
                     }
+                    if (value.hasMarkdown) {
+                        data[name + ':hasMarkdown'] = value.hasMarkdown;
```

Comment:
It will only ever be true or false, and I don't imagine that will change

https://github.com/dimagi/Vellum/blob/master/src/javaRosa.js#L51
https://github.com/dimagi/Vellum/blob/master/src/javaRosa.js#L1089